File: extensions/jashing-cvs/src/main/java/com/github/avarabyeu/jashing/integration/vcs/VCSClientException.java
package com.github.avarabyeu.jashing.integration.vcs;

/**
 * VCS clients exceptions
 *
 * @author Andrey Vorobyov
 */
public class VCSClientException extends RuntimeException {

    public VCSClientException(String message) {
        super(message);
    }

    public VCSClientException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: extensions/jashing-cvs/src/main/java/com/github/avarabyeu/jashing/integration/vcs/VCSClientException.kt
package com.github.avarabyeu.jashing.integration.vcs

/**
 * VCS clients exceptions

 * @author Andrey Vorobyov
 */
class VCSClientException : RuntimeException {

    constructor(message: String) : super(message) {
    }

    constructor(message: String, cause: Throwable) : super(message, cause) {
    }
}
